src/metabase/api/card.clj

```diff
@@ -516,14 +516,14 @@ saved later when it is ready."
   transaction and work in the `:card-create` event cannot proceed because the cards would not be visible outside of
   the transaction yet. If you pass true here it is important to call the event after the cards are successfully
   created."
-  ([card] (create-card! card false))
-  ([{:keys [dataset_query result_metadata dataset parameters parameter_mappings], :as card-data} delay-event?]
+  ([creator card] (create-card! creator card false))
+  ([creator {:keys [dataset_query result_metadata dataset parameters parameter_mappings], :as card-data} delay-event?]
```

Comment:
feels like the card-data should be either first or last, not in the middle of the params

Reply:
That's fair I'll swap the arg order to make it come first.